def raise_exception_msg(message=""):
